source/imaer-gml/src/test/resources/gml/v6_0/roundtrip/coldstart_without_characteristics.gml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<imaer:FeatureCollectionCalculator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:imaer="http://imaer.aerius.nl/6.0" gml:id="NL.IMAER.Collection" xsi:schemaLocation="http://imaer.aerius.nl/6.0 https://imaer.aerius.nl/6.0/IMAER.xsd">
+    <imaer:metadata>
+        <imaer:AeriusCalculatorMetadata>
+            <imaer:project>
+                <imaer:ProjectMetadata>
+                    <imaer:year>2030</imaer:year>
+                    <imaer:name>Cold start sources</imaer:name>
+                </imaer:ProjectMetadata>
+            </imaer:project>
+            <imaer:situation>
+                <imaer:SituationMetadata>
+                    <imaer:name>Cold start sources</imaer:name>
+                    <imaer:reference>ABCD1234</imaer:reference>
+                    <imaer:situationType>PROPOSED</imaer:situationType>
+                </imaer:SituationMetadata>
+            </imaer:situation>
+            <imaer:version>
+                <imaer:VersionMetadata>
+                    <imaer:aeriusVersion>DEV</imaer:aeriusVersion>
+                    <imaer:databaseVersion>DEV_DB</imaer:databaseVersion>
+                </imaer:VersionMetadata>
+            </imaer:version>
+        </imaer:AeriusCalculatorMetadata>
+    </imaer:metadata>
+    <imaer:featureMember>
+        <imaer:ColdStartEmissionSource sectorId="3150" gml:id="ES.1">
```

Comment:
Shouldn't this be sectorId `3160`?

Reply:
Maybe, the value doesn't matter at the moment though, as it is not hardcoded in the IMAER part for any decision.

The implementation lets the user free in specifying source characteristics or not. In the case of most sources, this means that characteristics in GML will be used if specified, and the sector default will be used if it wasn't specified. There are a few source types where characteristics will just be ignored if supplied, no matter if it was specified or not. ColdStart is new in the sense that characteristics will be read if present, but if they're not there then we won't fall back to sector defaults.

We could add another flag to cold start specifically, which would tell us if characteristics should be user defined or not, but not specifying it implicitly states the same?

Reply:
As discussed, added a flag on the emission source level.